araproc/analysis/rpr.py

```diff
@@ -0,0 +1,206 @@
+import numpy as np
+from scipy.ndimage import uniform_filter1d
+from araproc.framework import waveform_utilities as wfu
+
+
+def get_max_info(channel_wf, channel_time, use_debug=False):
+    """
+    Identifies the maximum voltage and its corresponding time.
+
+    Parameters
+    ----------
+    channel_wf: numpy array
+        Array of voltage values (squared waveform data).
+    channel_time: numpy array
+        Array of time values corresponding to the waveform data.
+    use_debug: bool, optional
+        If True, prints debug information.
+
+    Returns
+    -------
+    max_bin: int
+        Index of the maximum voltage in the waveform.
+    max_val: float
+        Maximum voltage value in the waveform.
+    max_time: float
+        Time corresponding to the maximum voltage value.
+    """
+    max_bin = np.nanargmax(channel_wf)
+    max_val = channel_wf[max_bin]
+    max_time = channel_time[max_bin]
+
+    if use_debug:
+        print(f"Max value: {max_val}, Max time: {max_time}")
+
+    return max_bin, max_val, max_time
+
+def get_sqrt_volt_sum_wf(channel_wf, dt):
```

Comment:
Maybe we can just put this function into `get_ch_sliding_v2_snr_uw`? It's pretty short and only used once.